providers/edge/src/airflow/providers/edge/plugins/templates/edge_worker_hosts.html

```diff
@@ -97,6 +100,25 @@ <h2>Edge Worker Hosts</h2>
                 {% endfor %}
               </ul>
             </td>
+            <td>
+              {%- if host.state == "idle" or host.state == "running" -%}
+                <form action="../edgeworkerhosts/status/maintenance/{{ host.worker_name }}/on" method="GET">
```

Comment:
Can we switch to POST, then we have less of a problem that an admin is having a Cross-Site-Scripting problem because CORS is checked.